README.md

```diff
@@ -137,7 +165,15 @@ def transform_response_date(db_model: BaseModel) -> Callable:
 You can optionally pass a cache object to the CRUDBuilder class. Right now only Memcached is supported, and only the pymemcached client
 has been tested. See https://pymemcache.readthedocs.io/en/latest/getting_started.html for more information on how to set up a pymemcached client.
 
-In theory any client with the methods:
+#### Caching implementation
+The way caching has been implemented, in the read-one case it relies upon the primary key of the model in question, and takes into account query parameters like equals_field, equals_value, relationships and sorting.
```

Comment:
Hmm, can you wordsmith this? Not sure it reads quite right to me. Specifically using a declarative statement with "The way caching..." followed by the "in the read-one case..." preposition.